Common/Notifications/Notification.cs

```diff
@@ -119,10 +124,15 @@ public class NotificationEmail : Notification
         /// <param name="data">Data to attach to the email</param>
         public NotificationEmail(string address, string subject, string message, string data)
         {
-            Message = message;
-            Data = data;
-            Subject = subject;
+            if (!Validate.EmailAddress(address))
+            {
+                throw new ArgumentException($"Invalid email address: {address}");
+            }
+
             Address = address;
+            Data = data ?? string.Empty;
+            Message = message ?? DefaultMessageAndSubject;
+            Subject = subject ?? DefaultMessageAndSubject;
```

Comment:
Maybe it's worth documenting this new behavior in the methods xml docs? If null will use `DefaultMessageAndSubject/string.Empty`

Reply:
Agreed. Notify docs should be updated as well as this constructor's documentation.